src/components/events/event-carousel.svelte

```diff
@@ -27,6 +35,12 @@
     carouselRef.addEventListener("mousemove", scrollOnMouseMove);
     carouselRef.addEventListener("mousedown", startGrabbing);
     carouselRef.addEventListener("mouseup", endGrabbing);
+    carouselButtonLeft.addEventListener("click", () => {
+      smoothScrollCarousel(-150);

```

Comment:
I think it might be worth it to turn this into a variable to be used in the CSS section as well, just so it's easier to be changed.

For reference: https://svelte.dev/repl/4b1c649bc75f44eb9142dadc0322eccd?version=3.6.7.